plugins/provisioners/ansible/cap/guest/oraclelinux/ansible_install.rb

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2020, Oracle and/or its affiliates.
+# Licensed under the MIT License.
+
+require_relative "../facts"
+require_relative "../pip/pip"
+
+module VagrantPlugins
+  module Ansible
+    module Cap
+      module Guest
+        module OracleLinux
+          module AnsibleInstall
```

Comment:
it would be sweet if this module gets a few tests, similar to https://github.com/hashicorp/vagrant/blob/master/test/unit/plugins/provisioners/ansible/cap/guest/alpine/ansible_install_test.rb